leaderboard/templates/leaderboard/updates.html

```diff
@@ -27,6 +27,7 @@
         <h2>Competition updates</h2>
         <p>
           <ul>
+            <li><em>11/6/2021:</em> Script for converting WMT XML files can be found here: <a href="https://github.com/wmt-conference/wmt-format-tools.git">https://github.com/wmt-conference/wmt-format-tools.git</a></li>

```

Comment:
What do you think about being more explicit, for example "Scripts for unwrapping and wrapping WMT XML files"?

Reply:
good idea